Assets/MixedRealityToolkit.Tests/PlayModeTests/PointerProfileTests.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Input;
+using NUnit.Framework;
+using UnityEditor;
+
+namespace Microsoft.MixedReality.Toolkit.Tests
+{
+    class PointerProfileTests
+    {
+        private const string eyeTrackingConfigurationProfileGuid = "6615cacb3eaaa044f99b917186093aeb";
```

Comment:
nit: can you add a comment that translates that guid to the path - when kurtis replaced our hard coded paths to guids he left the paths as comments so ppl could still figure out where those assets are